pkg/credentials/certchain.go

```diff
@@ -4,13 +4,13 @@ import (
 	"os"
 )
 
-const (
+var (
```

Comment:
Why change to var?

Reply:
to allow each binary to change them during the early init phase, like parsing flags. This could be dirty, but is the easiest and more consistent way avoiding to pass additional parameters where ever those credentials are used

Reply:
https://github.com/dapr/dapr/search?q=credentials.IssuerCertFilename

maybe not many places, but with the var is easier and no other change in the code.

Reply:
It is fine for now since they are all independent services.